service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterComponentsMapper.java

```diff
@@ -183,7 +183,10 @@ private String mapListResourceToComponents(ListResource listReferencedToEncounte
     }
 
     private Optional<String> mapProcedureRequest(Resource resource) {
-        return diaryPlanStatementMapper.mapDiaryProcedureRequestToPlanStatement((ProcedureRequest) resource, IS_NESTED);
+        return Optional.ofNullable(
+            diaryPlanStatementMapper.mapDiaryProcedureRequestToPlanStatement((ProcedureRequest) resource,
+            IS_NESTED)
```

Comment:
Having this formatted like tells me that `IS_NESTED` is a 2nd parameter of `ofNullable`.
Move it so it's all on line 188